src/components/Card.js

```diff
@@ -19,10 +24,20 @@ const Wrapper = styled.section`
   padding: 0.5rem;
   color: #394a59;
   gap: 0.75rem;
+  word-break: break-word;
+`;
+
+const Categories = styled.ul`
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: start;
```

Comment:
`start` is the default value of `justify-content`. Therefore, you don't need to declare this property.